### DISCUSSION TITLE
Music wont play

### DISCUSSION THREAD

**Discussion by Username2120** [HaxeFlixel/flixel#2784] [Q&A] [answered]
For some reason music refuses to play at all, no matter what code i use to get it to play.
I've tried:
`FlxG.sound.playMusic("assets/music/title-theme.ogg", 0.7, false);`
As well as
`FlxG.sound.playMusic(AssetPaths.title_theme__ogg, 0.7, false);`
Both of which I saw in some tutorial videos I watched because of this issue.
I've been using HaxeFlixel for a while and this is the only issue that still messes with me. Any ideas?
(And no my game isnt muted)

**Comment by bwkam** [comment]
- Does `FlxG.sound.play("assets/music/title-theme.ogg", 0.7, false);` work? Make sure that the name of your file is valid too.
- Does the console output any related error messages?

**Reply by Username2120** [reply]
For some reason when using this code the game boots up like normal without any music playing, but when I attempt to return to the title screen in anyway it gives me a null object reference, which it never did before. and while using the old code the console never printed anything. Any ideas?

**Reply by bwkam** [reply]
> but when I attempt to return to the title screen in anyway it gives me a null object reference

That's most likely not related to the music problem. 

The code you sent in the original post must work according to the [docs](https://api.haxeflixel.com/flixel/system/frontEnds/SoundFrontEnd.html#playMusic), something has to be wrong with the rest of your game logic.

**Reply by Username2120** [reply]
But that's exactly why I'm confused. Because when I revert back to the code I posted above, it no longer throws the null object reference. and no matter what I try, whether it be from the docs or a tutorial video from the Haxe website, the song never plays.

**Reply by Geokureli** [reply]
can you share more code?

**Reply by Geokureli** [reply]
did i accidentally close this issue or did you close this issue, @Username2120 ?

**Reply by moxie-coder** [reply]
> did i accidentally close this issue or did you close this issue, @Username2120 ?

he closed it

**Reply by Username2120** [reply, marked as answer]
Hi, sorry. i closed this myself because i had forgot about it after i solved the issue. turns out the audio files i was trying to use were corrupted... oops. its all solved now, sorry for the confusion. 
> did i accidentally close this issue or did you close this issue, @Username2120 ?
